src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingServiceTest.java

```diff
@@ -173,6 +173,9 @@ class FilingServiceTest {
     @Mock
     private OverseasEntityChangeService overseasEntityChangeService;
 
+    @Mock
+    private ManagingOfficerCessationService managingOfficerCessationService;
```

Comment:
Can you update the test/tests that verify this service is called please?

Reply:
Done.